src/backend/core/tests/test_models_invitations.py

```diff
@@ -73,6 +73,35 @@ def test_models_invitations__is_expired(settings):
     assert expired_invitation.is_expired is True
 
 
+def test_models_invitation__new_user__convert_invitation_to_access():
+    """
+    Upon creating a new identity, invitations linked to that email
+    should be converted to accesses and then deleted.
+    """
+    invitation = factories.InvitationFactory()
+    new_identity = factories.IdentityFactory(is_main=True, email=invitation.email)
+
```

Comment:
I would create some other invitation to make sure only the one with a matching email is converted